_includes/assets/js/search.js

```diff
@@ -1,115 +1,118 @@
-var indicatorSearch = function(inputElement, indicatorDataStore) {
-  that = this;
-  this.inputElement = inputElement;
-  this.indicatorDataStore = indicatorDataStore;
-  this.indicatorData = [];
-  this.hasErrored = false;
-
-  this.processData = function(data) {
-    for(var goalLoop = 0; goalLoop < data.length; goalLoop++) {
-      for(var indicatorLoop = 0; indicatorLoop < data[goalLoop].goal.indicators.length; indicatorLoop++) {
-        var currentIndicator = data[goalLoop].goal.indicators[indicatorLoop];
-        currentIndicator.goalId = data[goalLoop].goal.id;
-        currentIndicator.goalTitle = data[goalLoop].goal.title;
-        that.indicatorData.push(currentIndicator);
-      }
+var indicatorSearch = function() {
+  // Helper function to make a search query "fuzzier", using the ~ syntax.
+  var getFuzzierQuery = function(query, fuzziness) {
```

Comment:
Consider renaming `fuzziness` to `amountOfFuzziness` to make it clearer that's it's a positive integer. Of course "fuzziness" as a concept is a bit confusing at first glance until you read the Lunr docs which makes things pretty clear.